src/components/text-input.components.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable default-param-last */
+/* eslint-disable unicorn/better-regex */
+import $ from 'jquery';
+
+export default class TextInput {
+    constructor(stringQuery, textEnable = false, required = true, textValid = '') {
+        this.stringQuery = stringQuery;
+
+        this.textInput = $(`${stringQuery} input[type="text"]`);
+
+        this.textValid = textValid;
+        this.textEnable = textEnable;
+        this.required = required;
+    }
+
+    getTextInput() {
+        return this.textInput.val();
+    }
+
+    setTextInput(value) {
+        this.textInput = this.textInput.val(value);
+    }
+
+    getTextValid() {
+        return this.textValid.val();
```

Comment:
why do these methods do the same thing?
![image](https://user-images.githubusercontent.com/23269934/162942241-63314b73-8e01-4c7c-bf5a-a2fc5c6cf969.png)

Reply:
Fixed